src/test/kotlin/apply/application/MissionServiceTest.kt

```diff
@@ -50,4 +56,54 @@ class MissionServiceTest {
         every { missionRepository.existsByEvaluationId(any()) } returns true
         assertThrows<IllegalStateException> { missionService.save(missionData) }
     }
+
+    @Test
+    fun `특정 모집의 모든 과제를 찾는다`() {
+        val recruitmentId = 1L
+        val firstEvaluation = createEvaluation(id = 1L, title = "평가1", recruitmentId = recruitmentId)
+        val secondEvaluation = createEvaluation(id = 2L, title = "평가2", recruitmentId = recruitmentId)
+        every { evaluationRepository.findAllByRecruitmentId(any()) } returns listOf(firstEvaluation, secondEvaluation)
+
+        val firstMission = createMission(title = "과제1", evaluationId = 1L)
+        val secondMission = createMission(title = "과제1", evaluationId = 2L)
+        every { missionRepository.findAllByEvaluationIdIn(any()) } returns listOf(firstMission, secondMission)
+
+        val actual = missionService.findAllByRecruitmentId(recruitmentId)
+        assertAll(
+            { assertThat(actual).hasSize(2) },
+            {
+                assertThat(actual).containsAnyOf(
+                    MissionResponse(firstMission, firstEvaluation),
+                    MissionResponse(secondMission, secondEvaluation)
+                )
+            }
+        )
+    }
+
+    @Test
+    fun `과제를 삭제한다`() {
+        val missionId = 1L
+        every { missionRepository.findByIdOrNull(missionId) } returns createMission(submittable = false)
+        every { missionRepository.deleteById(any()) } just Runs
+
+        assertDoesNotThrow { missionService.deleteById(missionId) }
```

Comment:
```suggestion
        val mission = createMission(submittable = false)
        every { missionRepository.findByIdOrNull(any()) } returns mission
        every { missionRepository.deleteById(any()) } just Runs

        assertDoesNotThrow { missionService.deleteById(mission.id) }
```

테스트들에서 변수로 mission을 두고 사용하면 더 명확하게 표혈할 수 있을 것 같은데 이런 스타일은 어떠신가요 ?
그냥 참고만 부탁드려요 ㅎㅎ
(RecruitmentServiceTest를 참고했습니다)